atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/MultiCassandraStartupOrderingEteTest.java

```diff
@@ -68,56 +69,45 @@ public void randomizeKeyspace() throws IOException, InterruptedException {
                 + "/' var/conf/atlasdb-ete.yml");
     }
 
-    @AfterClass
-    public static void resetKeyspace() throws IOException, InterruptedException {
-        EteSetup.execCliCommand("sed -i 's/keyspace: .*/keyspace: atlasete/' var/conf/atlasdb-ete.yml");
-        stopTheAtlasServer();
-        startTheAtlasServer();
+    @After
+    public void restoreCassandra() throws InterruptedException {
+        startCassandraNodes();
     }
```

Comment:
It seems unexpected that we wouldn't need to reset the keyspace, though I guess the sed runs within the docker container and we don't reuse ETE-server containers.